README.md

```diff
@@ -4,7 +4,7 @@
 
 Clone this repository and navigate to the directory:
 ```
-git clone git@github.com:Sage-Bionetworks/genie-bpc-pipeline.git
+git clone https://github.com/Sage-Bionetworks/genie-bpc-pipeline.git
```

Comment:
nit: One thing to note here.  the `git@` is not technically wrong, some people just use `ssh` keys to connect with github.